CHANGELOG.md

```diff
@@ -63,7 +63,7 @@
 - PR #645 Fix cmake build type handling not setting debug options when CMAKE_BUILD_TYPE=="Debug"
 - PR #665 Reworked the hash map to add a way to report the destination partition for a key
 - PR #670 CMAKE: Fix env include path taking precedence over libcudf source headers
-
+- PR #674 Check for gdf supported column types
```

Comment:
We have two line breaks between versions, one line break between sections within a version.
```suggestion
- PR #674 Check for gdf supported column types

```